external/llvm-project/mlir/include/mlir/Dialect/AMDGPU/Utils/Chipset.h

```diff
@@ -19,7 +19,7 @@ struct Chipset {
   static FailureOr<Chipset> parse(StringRef name);
 
   bool isGfx940() const {
```

Comment:
are we sure some arch in the future won't have support for E5M2FNUZ and E4M3FNUZ? If that happens we will have to change the name of the function to something else.